server/src-lib/Hasura/RQL/DDL/Metadata/Generator.hs

```diff
@@ -48,17 +43,26 @@ genReplaceMetadata = do
     <*> arbitrary
     <*> arbitrary
   where
-    genFunctionsMetadata :: MetadataVersion -> Gen FunctionsMetadata
+    genFunctionsMetadata :: MetadataVersion -> Gen Functions
     genFunctionsMetadata = \case
-      MVVersion1 -> FMVersion1 <$> arbitrary
-      MVVersion2 -> FMVersion2 <$> arbitrary
+      MVVersion1 -> OM.fromList . map (\qf -> (qf, FunctionMetadata qf emptyFunctionConfig)) <$> arbitrary
+      MVVersion2 -> arbitrary
+
+instance (Arbitrary k, Eq k, Hashable k, Arbitrary v) => Arbitrary (InsOrdHashMap k v) where
```

Comment:
Having those instances here seem.... arbitrary :drum: 
Jokes aside: I think we could move those to a dedicated module... maybe `Hasura.Generators` or something? But that could be done in a separate PR.

Reply:
Let me at least add a docstring/comment to this module suggesting this.